web/src/pages/index.md

```diff
@@ -7,36 +7,29 @@ title: ECE 477 Team 5 Homepage
 
 ## Project Description
 
-The project at hand involves the development of a smart air hockey table that
-replicates the functionalities of a traditional arcade air hockey setup with a
-twist. The notable innovation is the incorporation of a responsive surface
-beneath a translucent playfield. This reactive surface, comprising strategically
-positioned lights, sensors, and computational components, reacts in real-time to
-the movements of the puck, generating visual displays that enhance the gameplay
-experience.
-
-The dynamic nature of the reactive surface introduces novel game modes that
-transcend the boundaries of conventional air hockey. Despite its technologically
-advanced features, the smart air hockey table maintains a compact form factor,
-making it adaptable to a variety of flat surfaces, from regular tables to other
-suitable platforms. This convergence of timeless gameplay with modern
-technological advancements is poised to redefine the enjoyment of air hockey,
-offering enthusiasts an immersive and unparalleled gaming encounter.
+The Smart Air Hockey Table replicates the functionalities of a traditional arcade air hockey setup with a twist. The notable innovation is the incorporation of a responsive surface beneath a translucent playfield. This reactive surface, comprising strategically positioned lights, sensors, and computational components, reacts in real-time to the movements of the puck, generating visual displays that enhance the gameplay experience.
+
+A tuned grid of Hall effect sensors lies beneath the playing field of the Smart Air Hockey Table. These sensors measure disturbances in the magnetic field caused by a magnet embedded within the puck. Microcontrollers within the table read the values generated by these Hall effect sensors and process the data to get an accurate position of the puck in real-time.
```

Comment:
These sensors measure disturbances in the magnetic field caused by a magnet embedded within the puck.